src/views/CountryRiskWatch/index.tsx

```diff
@@ -1,8 +1,99 @@
+import ReturnPeriodTable from './ReturnPeriodTable';
+import HistoricalDataChart from './HistoricalDataChart';
+
+import i18n from './i18n.json';
+import styles from './styles.module.css';
+import RiskBarChart from './RiskBarChart';
+
+type GetCountrySeasonal = paths['/api/v1/country-seasonal/']['get'];
+// FIXME: query type not available
+// type CountryRiskQuery = GetCountryRisk['parameters']['query'];
+type CountrySeasonal = GetCountrySeasonal['responses']['200']['content']['application/json'];
+
 // eslint-disable-next-line import/prefer-default-export
 export function Component() {
+    const { countryResponse } = useOutletContext<CountryOutletContext>();
+    const { countryId } = useParams<{ countryId: string }>();
+    const strings = useTranslation(i18n);
+    const [
+        selectedMonths,
+        setSelectedMonths,
+    ] = useInputState<Record<number, boolean> | undefined>({ 0: true });
+
+    const {
+        pending: pendingCountryRiskResponse,
+        response: countryRiskResponse,
+    } = useRequest<CountrySeasonal>({
+        apiType: 'risk',
+        url: '/api/v1/country-seasonal/',
+        query: {
+            iso3: countryResponse?.iso3?.toLowerCase(),
+        },
+    });
+
     return (
-        <div>
-            This is country Risk Watch
+        <div className={styles.countryRiskWatch}>
+            <Container
+                heading={strings.risksByMonthHeading}
+                headingLevel={2}
+                headerDescription={strings.risksByMonthDescription}
+                withHeaderBorder
+                childrenContainerClassName={styles.riskByMonthContent}
+            >
+                <MultiMonthSelectInput
+                    name={undefined}
+                    value={selectedMonths}
+                    onChange={setSelectedMonths}
+                />
+                <RiskTable
+                    riskData={countryRiskResponse?.[0]}
+                    selectedMonths={selectedMonths}
+                    dataPending={pendingCountryRiskResponse}
+                />
+            </Container>
+            <div className={styles.eapSection}>
+                <Container
+                    className={styles.eapContainer}
+                    heading={strings.eapHeading}
+                    withHeaderBorder
+                    actions={(
+                        <Link
+                            to="https://www.ifrc.org/appeals?date_from=&date_to=&type%5B%5D=30&appeal_code=&text="
+                            withExternalLinkIcon
+                            variant="primary"
+                        >
+                            {strings.eapDownloadButtonLabel}
+                        </Link>
+                    )}
+                >
+                    {strings.eapDescription}
+                </Container>
+            </div>
+            <RiskBarChart
+                pending={pendingCountryRiskResponse}
+                riskData={countryRiskResponse?.[0]}
```

Comment:
Why are we only using the first data point? Let's add a NOTE

Reply:
We only get 1 data point

Reply:
Let's add a NOTE for now